javaslang/src/test/java/javaslang/collection/AbstractTraversableTest.java

```diff
@@ -1048,6 +1048,9 @@ public void shouldReturnNoneWhenComputingMaxOfNil() {
     }
 
     @Test
+    public void shouldComputeMaxOfOneValue() { assertThat(Iterator.of(5).max()).isEqualTo(Option.some(5)); }
```

Comment:
Please change `Iterator.of(...)` to just `of(...)`.

AbstractTraversableTest has a protected method [of(T)](https://github.com/javaslang/javaslang/blob/51cb15c56e6347b39a7fbedd222f02a8e967701e/javaslang/src/test/java/javaslang/collection/AbstractTraversableTest.java#L46) that is implemented by all Test classes, [especially by `IteratorTest`](https://github.com/javaslang/javaslang/blob/51cb15c56e6347b39a7fbedd222f02a8e967701e/javaslang/src/test/java/javaslang/collection/IteratorTest.java#L101-L104).

Thank you!